src/components/Composer.vue

```diff
@@ -69,6 +69,7 @@
 							<ListItemIcon :no-margin="true"
 								:name="option.label"
 								:subtitle="option.email"
+								:iconClass="!option.id ? 'icon-user' : null"
```

Comment:
I am not sure if icon classes are still supported. I remember that we wanted to get rid of them an replace with inline (material) icons.

icon-class is not documented at https://nextcloud-vue-components.netlify.app/#/Components/NcListItems?id=nclistitemicon either

Reply:
> icon-class is not documented at 

`iconClass` is a valid prop of `NcAvatar` https://nextcloud-vue-components.netlify.app/#/Components/NcAvatar and will be thrown via attrs from `NcListItems`.

In current solution of `NcListItems` it is not possible to use material icons instead. Vue component need to be adjusted + NcVue release is needed.